lighthouse-core/audits/byte-efficiency/byte-efficiency-audit.js

```diff
@@ -62,14 +62,13 @@ class UnusedBytes extends Audit {
    * @return {!Promise<!AuditResult>}
    */
   static audit(artifacts) {
-    const devtoolsLogs = artifacts.devtoolsLogs[Audit.DEFAULT_PASS];
-    return artifacts.requestNetworkRecords(devtoolsLogs).then(networkRecords => {
-      return artifacts.requestNetworkThroughput(networkRecords).then(networkThroughput =>
-        Promise.resolve(this.audit_(artifacts, networkRecords)).then(result =>
-          this.createAuditResult(result, networkThroughput)
-        )
-      );
-    });
+    const devtoolsLog = artifacts.devtoolsLogs[Audit.DEFAULT_PASS];
+    return artifacts.requestNetworkRecords(devtoolsLog)
+      .then(networkRecords => this.audit_(artifacts, networkRecords))
+      .then(result => {
```

Comment:
awkwardness like this due to needing to hold onto variables will be a lot better once we can use async/await